source/adapters/cuda/image.cpp

```diff
@@ -311,7 +311,13 @@ ur_result_t urTextureCreate(ur_sampler_handle_t hSampler,
     ur_exp_sampler_cubemap_filter_mode_t CubemapFilterModeProp =
         hSampler->getCubemapFilterMode();
     if (CubemapFilterModeProp == UR_EXP_SAMPLER_CUBEMAP_FILTER_MODE_SEAMLESS) {
+#if CUDA_VERSION >= 11060
       ImageTexDesc.flags |= CU_TRSF_SEAMLESS_CUBEMAP;
+#else
+      setErrorMessage("The "UR_EXP_SAMPLER_CUBEMAP_FILTER_MODE_SEAMLESS" feature requires cuda 11.6 or later.",
+                        UR_RESULT_ERROR_ADAPTER_SPECIFIC);
+      return UR_RESULT_ERROR_ADAPTER_SPECIFIC;
```

Comment:
I think `$X_RESULT_ERROR_ADAPTER_SPECIFIC` needs to be added to the `returns:` part of `SampledImageCreateExp` and `UnsampledImageCreateExp` in `scripts/core/exp-bindless-images.yml`.

Reply:
Thanks, this is done.